data/mods/Magiclysm/monsters/zombified_monsters.json

```
@@ -258,5 +258,21 @@
     "special_attacks": [ { "id": "stretch_bite", "cooldown": 1, "range": 2 } ],
     "flags": [ "SEES", "HEARS", "SMELLS", "KEENNOSE", "WARM", "STUMBLES", "BASHES", "POISON", "NO_BREATHE", "REVIVES" ],
     "armor": { "bash": 4, "cut": 2, "necrotic": 75, "poison": 30 }
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_giant_heron_ridable_zombie",
+    "name": {
+      "//~": "Word is invented: meaning is given as 'white-striders' ",
+      "str": "zombified aenor",
+      "str_pl": "zombified aenóren"
+    },
+    "description": "With bedraggled feathers and rent skin, this .",
```

```suggestion
    "description": "With bedraggled feathers and rent skin, this .",
```
Description unfinished.